lib/screens/landing_page/landing_page.dart

```diff
@@ -0,0 +1,216 @@
+import 'package:flutter/material.dart';
+import 'package:promdi_fe/screens/login/login.dart';
+import 'package:promdi_fe/screens/signup/signup_page.dart';
+import 'package:promdi_fe/widgets/card.dart';
+
+class LandingPage extends StatefulWidget {
+  const LandingPage({Key? key}) : super(key: key);
+
+  @override
+  _LandingPageState createState() => _LandingPageState();
+}
+
+class _LandingPageState extends State<LandingPage> {
+  @override
+  Widget build(BuildContext context) {
+    final height = MediaQuery.of(context).size.height;
+    return Scaffold(
+        appBar: AppBar(
+          centerTitle: true,
+          title: Text('Promdi App'),
+          leading: ClipOval(
+            child: Image.asset(
+              'assets/icons/logo.png',
+              fit: BoxFit.cover,
+              height: height * 0.04,
+            ),
+          ),
+        ),
+        body: SingleChildScrollView(
+          child: Column(
+            children: [
+              Card(
+                color: Colors.transparent,
+                child: Padding(
+                  padding: EdgeInsets.all(10),
+                  child: Row(
+                    mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                    children: [
+                      Image.asset(
+                        'assets/images/cloudy.png',
+                        height: height * 0.1,
+                      ),
+                      Column(
+                        children: [
+                          Text(
+                            '25c',
+                            style: TextStyle(
+                                fontWeight: FontWeight.bold, fontSize: 20),
+                          ),
+                          Text('Monday 9th Aug 2021'),
```

Comment:
Can we have the actual date, make use of the library functions to get the device native date.

Reply:
done

Reply:
Let's have the date in the format Aug 10th, 2021